xcode/Subconscious/Subconscious.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -5,8 +5,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/twostraws/CodeScanner",
       "state" : {
-        "revision" : "d8f10d992ef32a4187b78a6df320facf75d631d8",
-        "version" : "2.3.1"
+        "revision" : "4b2f9c57d03bfe94e6b45572e0c908c1433c32b2",
+        "version" : "2.3.2"
```

Comment:
Intentional? I'm fine with updating it but just flagging because it's unrelated.

Reply:
I believe this must have happened automatically while updating Noosphere. We aren't constraining the CodeScanner version tightly.

Would prefer to leave as is, and file a follow-up to constrain dependencies as appropriate.